test/content-security-policy.test.ts

```diff
@@ -372,9 +372,7 @@ describe("Content-Security-Policy middleware", () => {
       contentSecurityPolicy({
         directives: {},
       });
-    }).toThrow(
-      /^Content-Security-Policy has no directives. Either set some or disable the header$/
-    );
+    }).not.toThrow();
```

Comment:
I think this test should continue to `throw`, which means we'll need to set `useDefaults: false`.

Reply:
Fixed in https://github.com/helmetjs/helmet/pull/316/commits/2c497fbb1dd47187852aee1dc51008d40958ade4